examples/103-autoscaling-pipeline.py

```diff
@@ -8,25 +8,26 @@
 Replicas are automatically scaled up and down depending on the the desired formula, which can be computed using the following:
 
 * `P` total number of pending messages.
-* `p` change in number of pending messages
+* `p` change in number of pending messages.
 * `c` the current number of replicas.
-* `minmax(v, min, max)` a function to constraint the minimum and maximum number of replicas.
+* `limit(v, min, max, delta)` a function to constraint the minimum and maximum number of replicas, as well as the step-up/down.
 
 In this example:
 
-* Aa period is 60s
-* Each replica can consume 250 messages each second
-* We want to consume all pending messages in 10 periods. 
+* Each period is 60s.
```

Comment:
I would add here that period = scalingDelay, and explicitly say that scalingDelay means how often scale formula will be executed. Maybe it's just me, but it took me a moment to understand this documentation and figure out what all those constants mean.

Reply:
I think this needs some better docs all in all.